example/readFromString/readFromString.cpp

```diff
@@ -0,0 +1,35 @@
+#include "json/json.h"
+/*
+parse a string to Value object with CharReaderBuilder class or Reader class
+>g++ readFromString.cpp -ljsoncpp -std=c++11 -o readFromString
+>./readFromString
+colin
+20
+*/
+int main() {
+  std::string strRes = "{\"Age\": 20, \"Name\": \"colin\"}";
+  int nLen = (int)strRes.length();
+  const char* pStart = strRes.c_str();
```

Comment:
we don't need pStart until the parse() call. Calculate it there or just use the expression directly in the parse call.